---
title: "<b>1\\.</b> 
<u>Qiaole Dong</u>, Chenjie Cao, and Yanwei Fu<sup><a title='Corresponding author'>✉</a></sup> (2013). **Pinda: A Web service for detection 
and analysis of intraspecies gene duplication events.** Computer Methods 
and Programs in Biomedicine 111(3):711-714."
collection: publications
permalink: /publication/01_Pinda_A_Web_service_for_detection
excerpt: '
  [<span class="underline-on-hover" style="color:#FF6F6F">Abstract</span>](../publication/01_Pinda_A_Web_service_for_detection)
\| [<span class="underline-on-hover" style="color:#FF6F6F">Full text</span>](http://www.sciencedirect.com/science/article/pii/S0169260713001788)
\| [<span class="underline-on-hover" style="color:#FF6F6F">Website</span>](http://orion.mbg.duth.gr/Pinda/)
\| [<span class="underline-on-hover" style="color:#FF6F6F">Source code</span>](https://github.com/dgkontopoulos/Pinda/)
\| [<span class="underline-on-hover" style="color:#FF6F6F"><img src="../images/bibtex.svg">citation</span>](../bibtex/1_Pinda.bib)
'
---

---
title: "<b>1\\.</b> 
<u>Qiaole Dong<\u>, Chenjie Cao, and Yanwei Fu<sup><a title='Corresponding author'>✉</a></sup>. 
**Incremental transformer structure enhanced image inpainting with masking positional encoding.** 
CVPR 2022. (* indicates co-first authour)"
collection: publications
permalink: /publication/1_CVPR2022_ZITS
excerpt: '
  [<span class="underline-on-hover" style="color:#FF6F6F">Abstract</span>](../publication/1_CVPR2022_ZITS)
\| [<span class="underline-on-hover" style="color:#FF6F6F">Full text</span>](https://openaccess.thecvf.com/content/CVPR2022/papers/Dong_Incremental_Transformer_Structure_Enhanced_Image_Inpainting_With_Masking_Positional_Encoding_CVPR_2022_paper.pdf)
\| [<span class="underline-on-hover" style="color:#FF6F6F">Data and code</span>](https://github.com/DQiaole/ZITS_inpainting)
'
---

<br><center><img src="../images/publications/pindalogo.png"></center> 

### Abstract:

<p style='text-align: justify;'>
We present Pinda, a Web service for the detection and analysis of 
possible duplications of a given protein or DNA sequence within a 
source species. Pinda fully automates the whole gene duplication 
detection procedure, from performing the initial similarity searches, 
to generating the multiple sequence alignments and the corresponding 
phylogenetic trees, to bootstrapping the trees and producing a 
Z-score-based list of duplication candidates for the input sequence. 
Pinda has been cross-validated using an extensive set of known and 
bibliographically characterized duplication events. The service 
facilitates the automatic and dependable identification of gene 
duplication events, using some of the most successful bioinformatics 
software to perform an extensive analysis protocol. Pinda will prove of 
use for the analysis of newly discovered genes and proteins, thus also 
assisting the study of recently sequenced genomes. The service's 
location is 
<a href='http://orion.mbg.duth.gr/Pinda'>http://orion.mbg.duth.gr/Pinda</a>. 
The source code is freely available via 
<a href='https://github.com/dgkontopoulos/Pinda/'>https://github.com/dgkontopoulos/Pinda/</a>.
</p>

[<span class="underline-on-hover" style="color:#FF6F6F">Full text</span>](http://www.sciencedirect.com/science/article/pii/S0169260713001788)
\| [<span class="underline-on-hover" style="color:#FF6F6F">Website</span>](http://orion.mbg.duth.gr/Pinda/)
\| [<span class="underline-on-hover" style="color:#FF6F6F">Source code</span>](https://github.com/dgkontopoulos/Pinda/)
\| [<span class="underline-on-hover" style="color:#FF6F6F"><img src="../images/bibtex.svg">citation</span>](../bibtex/1_Pinda.bib)
